src/App.js

```diff
@@ -94,22 +95,20 @@ class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          {this.state.user ? <HeaderNav user={this.state.user} /> : null}
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            <FormattedHTMLMessage id="app.title" />
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <FormattedHTMLMessage id="app.link" />
-          </a>
-          <p>{this.state.user ? this.state.user.Email : ''}</p>
-        </header>
+        {this.state.user ? <Header className="App-header" user={this.state.user} /> : null}
```

Comment:
At a glance I thought, maybe it is better moving this logic inside _Header_, but then I understand that when the user is not logged, probably even in the future, there will not be a header at all. 

Or maybe there will be a different header, so: What do you think about using a more specific name for current _Header_? maybe _AuthSessionHeader_ or something like that?

Reply:
Yes i can move inside Header component. Maybe its better